process/project_proposals.adoc

```diff
@@ -1,7 +1,92 @@
-== CNCF Project Proposal Process v1.2
+== CNCF Project Proposal Process v1.3
 
- . *Introduction*. This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
- . *Project Proposal Requirements*. Projects must be proposed via https://github.com/cncf/toc/tree/master/proposals[GitHub]. Project proposals submitted to the CNCF (see https://github.com/cncf/toc/blob/master/proposals/kubernetes.adoc[example]) can be written in https://www.markdownguide.org[Markdown], http://asciidoc.org[AsciiDoc], or http://docutils.sourceforge.net/rst.html[reStructuredText] and must provide the following information to the best of your ability:
+=== Introduction 
+
+This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
+
+=== Project stages
+
+image::project-stages.png[Project stages]
+
+=== Sandbox process
+
+All exceptions (and "no" outcomes) are handled by the TOC. Timeframes are approximate, to set expectations.
+
+image::sandbox-process.png[Sandbox process]
+
+. *Project Proposal*
+   * Project proposed through a https://github.com/cncf/toc/issues[GitHub issue]
+   * Template TBD
+. *TOC Triage* _2 weeks max_
+   * Performed by any TOC member
+   * Lazy TOC consensus before declining outright
```

Comment:
any chance this could be worded to convey "not yes" instead of "no"?  I know it sounds trivial, but "not yes" conveys that the project can come back at a later time when it is more mature.

Reply:
good idea!